packages/block-editor/src/components/global-styles/color-panel.js

```diff
@@ -228,15 +230,36 @@ function ColorPanelDropdown( {
 							{ 'is-open': isOpen }
 						),
 						'aria-expanded': isOpen,
+						ref: colorGradientDropdownButtonRef,
 					};
 
 					return (
-						<Button __next40pxDefaultSize { ...toggleProps }>
-							<LabeledColorIndicators
-								indicators={ indicators }
-								label={ label }
-							/>
-						</Button>
+						<>
+							<Button { ...toggleProps } __next40pxDefaultSize>
+								<LabeledColorIndicators
+									indicators={ indicators }
+									label={ label }
+								/>
+							</Button>
+							{ hasValue() && (
+								<Button
+									__next40pxDefaultSize
+									label={ __( 'Reset' ) }
+									aria-label={ sprintf(
+										/* translators: %s: The name of the color property being reset e.g. "Background" */
+										__( 'Reset %s' ),
+										label.toLowerCase()
+									) }
```

Comment:
When the reset button is focused, my screen reader (NVDA) reads two pieces of text.

```
Reset text  button
Reset
```

@tyxla Is there a way to keep the tooltip text as "Reset" but have the screen reader just read "Reset text"?

Reply:
> @tyxla Is there a way to keep the tooltip text as "Reset" but have the screen reader just read "Reset text"?

I think you should be able to do it by using the `description` prop instead of `aria-label`.

Reply:
I remembered that the approach of embedding dynamic strings as placeholders within a single sentence can be problematic when it comes to localization (See [this comment](https://github.com/WordPress/gutenberg/pull/65193#issuecomment-2419585680)).

How about simply using `Reset` here?

If that word alone is not enough, I think "Reset: %s" would also work. This format should not cause any problems when localizing. This approach is very popular.

https://github.com/search?q=repo%3AWordPress%2Fgutenberg+%22%3A+%25s%27+%29%22&type=code

Reply:
Matching `aria-label` and `Tooltip`'s text sounds good to me, and it will also avoid adding a redundant `aria-describedby` prop (see #65989)